docs/README.md

```diff
@@ -7,8 +7,8 @@ or hang out with our developers on either our [Slack](https://slack.rancher.io/)
 
 - Installation
   * [ISO Mode](./iso-installation.md)
-  * [App Mode - for development](./app-mode-installation.md)
   * [Automatic Mode - for mass installation](./automatic-installation.md)
+  * [App Mode - for development](./app-mode-installation.md)
```

Comment:
We removed the App mode and call it Dev mode recently(https://github.com/harvester/harvester/commit/3758022490af00442191ca3c58a8c911a4bf12ef).
We may also want to update that in Github issue templates